benchmarks/sort-cases.py

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env python3                                                         
+
+import time, argparse
+import numpy as np
+import arkouda as ak
+
+def is_cosorted(data):
+    # (b[0] > a[0]) | ((b[0] == a[0]) & recurse(a[1], b[1]))
+    def helper(x, right):
+        return (x[1:] > x[:-1]) | ((x[1:] == x[:-1]) & right)
+    right = ak.ones(data[0].size - 1, dtype=ak.bool)
+    for x in reversed(data):
+        right = helper(x, right)
+    return right.all()   
```

Comment:
Is this something that would be worth adding to arkouda as a counterpart to `ak.is_sorted`? I don't have a good sense of how often this would come up

Also I really like the way this function is implemented, it's super slick

Reply:
we could definitely make this perform a lot better if we did it on the server side, IMHO.